### ISSUE TITLE
Getting "Something went wrong" error when deleting added Role

### ISSUE THREAD

**Issue by NilukaSripalim** [wso2/product-is#12957]
**Describe the issue:**
When trying to delete an Added role, via console getting "Something went wrong" error

Note: This only happens when deleting a role that a user has already added. The user can either assign the role while creating or after the role has been created.

**How to reproduce:**
1. Add new Role via console
2. Assign an admin  user
3. Delete the role via UI (basic section)

**Expected behavior:**
The role should be deleted without any errors.

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Product Version: 5.12.0 alpha9
 - OS: Mac
 - Database: MySQL 5.7
 - Userstore: JDBC

---

![Screenshot 2022-01-10 at 22 37 07](https://user-images.githubusercontent.com/39120228/148811206-0603cfe1-905c-4c55-967e-7749c464fa95.png)

**Comment by piraveena** [comment]
I'm not getting the "Something went wrong" page. Getting this "Session Expired" page. Tried in both MySQL 8 and H2. Both are showing this page.
<img width="1544" alt="Screenshot 2022-03-24 at 11 17 40" src="https://user-images.githubusercontent.com/17597293/159850968-7993bc28-8cc6-4ea6-a710-b09b712aae07.png">

This page is rendered because, we have removed a role that is assigned for the logged-in user. Since there is a privilege change for the current logged-in user, we are revoking the access tokens issued for that user. This PR is sent to revoke the tokens when the user's privilege gets changed https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1656

This needs to be handled gracefully from the UI level. When the current user's role gets removed, we need to handle it gracefully or give some warning saying the current session will be removed.
